.github/CODEOWNERS

```diff
@@ -11,6 +11,9 @@
 # Core
 /sdk/core/ @pakrym @KrzysztofCwalina
 
+# Extensions
+/sdk/extensions @parkym
```

Comment:
I assume the lack of a trailing slash is intentional?

Reply:
Nope, thank you for noticing.